solidity/test/support/helpers.js

```diff
@@ -204,16 +190,102 @@ let _0x,
     return link.transferAndCall(oc.address, amount, args)
   }
 
-  _0x = (val) => {
-    return `0x${toHex(val)}`
+  newUint8ArrayFromStr = (str) => {
+    const codePoints = Array.prototype.map.call(str, c => c.charCodeAt(0))
+    return Uint8Array.from(codePoints)
+  }
+
+  // newUint8Array returns a uint8array of count bytes from either a hex or
+  // decimal string, hex strings must begin with 0x
+  newUint8Array = (str, count) => {
+    let result = new Uint8Array(count)
+
+    if (str.startsWith('0x') || str.startsWith('0X')) {
+      const hexStr = str.slice(2).padStart(count * 2, '0')
+      for (let i = result.length; i >= 0; i--) {
+        const offset = i * 2
+        result[i] = parseInt(hexStr[offset] + hexStr[offset + 1], HEX_BASE)
+      }
+    } else {
+      const num = bigNum(str)
+      result = newHash('0x' + num.toString(HEX_BASE))
+    }
+
+    return result
+  }
+
+  // newSignature returns a 65 byte Uint8Array for representing a signature
+  newSignature = (str) => {
+    return newUint8Array(str, 65)
   }
+
+  // newHash returns a 65 byte Uint8Array for representing a hash
+  newHash = (str) => {
+    return newUint8Array(str, 32)
+  }
+
+  // newAddress returns a 20 byte Uint8Array for representing an address
+  newAddress = (str) => {
+    return newUint8Array(str, 20)
+  }
+
+  // lengthTypedArrays sums the length of all specified TypedArrays
+  lengthTypedArrays = (...arrays) => {
+    return arrays.reduce((a, v) => a + v.length, 0)
+  }
+
+  toBuffer = (uint8a) => {
+    return Buffer.from(uint8a)
+  }
+
+  // concatTypedArrays recursively concatenates TypedArrays into one big
+  // TypedArray
+  // TODO: Does not work recursively
+  concatTypedArrays = (...arrays) => {
+    let size = lengthTypedArrays(...arrays)
+    let result = new arrays[0].constructor(size)
+    let offset = 0
+    arrays.forEach((a) => {
+      result.set(a, offset)
+      offset += a.length
+    })
+    return result
+  }
+
+  toHexWithoutPrefix = arg => {
+    if (arg instanceof Buffer) {
+      return arg.toString('hex')
+    } else if (arg instanceof Uint8Array) {
+      return Array.prototype.reduce.call(arg, (a, v) => a + v.toString('16').padStart(2, '0'), '')
+    } else {
+      return Buffer.from(arg, 'ascii').toString('hex')
+    }
+  }
+
+  toHex = (value) => {
+    return `0x${toHexWithoutPrefix(value)}`
+  }
+
+  splitRPCSignature = (oracleSignature) => {
+    let v = oracleSignature[64]
```

Comment:
For the future, it is possible for V to be more than one byte, but I'm fine waiting until we run into that problem to change this.